<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDijkstra</class>
 <widget class="QDialog" name="FrmDijkstra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rutas Dijkstra</string>
  </property>
  <widget class="QTableWidget" name="tbl_tableWidgetTX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <column>
    <property name="text">
     <string>ID_DF</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID_DEFAULT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID_FRAME(dec)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnEnviarRutas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>Send New
Routes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerNodosyPesos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nodes and weights</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerTodosDatosRecibidos">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View data received</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dijkstra Algorithm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbrPogressBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
